includes/base/widget-base.php

```diff
@@ -1225,24 +1225,6 @@ private function get_widget_css() {
 
 	}
 
-	/**
-	 * Widgets styles loading strategy.
-	 *
-	 * Which loading strategy to apply, either loading the widgets styles
-	 * in the `<head>` or in the `<body>`.
-	 *
-	 * Currently, it's based on the optimized CSS loading experiment. In
-	 * the future, will be replaced with a setting in the performance tab.
-	 *
-	 * @since 3.23.4
-	 * @access public
-	 *
-	 * @return bool Whether to load widgets styles in the `<head>`.
-	 */
-	public function load_widgets_styles_in_head(): bool {
```

Comment:
Just for clarification, why has this been removed?

Reply:
Elementor will no longer offer load styles in the `<body>`, only in the `<head>`. This should fix most of the Layout Shift issues and CSS specificity issues Elementor currently have. It will also improve the assets caching (ask @KingYes). And help us move away from the "Improved CSS Loading" experiment.